app/pages/settings/email.jsx

```diff
@@ -0,0 +1,315 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
+import apiClient from 'panoptes-client/lib/api-client';
+import talkClient from 'panoptes-client/lib/talk-client';
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+
+counterpart.registerTranslations('en', {
```

Comment:
This whole block should be moved to [the English locale file](/zooniverse/Panoptes-Front-End/blob/master/app/locales/en.js).